source/jormungandr/tests/routing_tests.py

```diff
@@ -85,6 +85,21 @@ def test_other_filtering(self):
         assert len(response['journeys']) == 1
         assert response['journeys'][0]["type"] == "non_pt_walk"
 
+    def test_speed_factor_direct_path(self):
+        """We test the coherence of the non pt walk solution with a speed factor"""
+
+        response = self.query_region("{query}&type=non_pt_walk&walking_speed=1.5".
+                                     format(query=journey_basic_query))
+
+        assert len(response['journeys']) == 1
+        assert response['journeys'][0]["type"] == "non_pt_walk"
+        assert len(response['journeys'][0]['sections']) == 1
+        print response['journeys'][0]['duration']
+        print response['journeys'][0]['sections'][0]['duration']
+        assert response['journeys'][0]['duration'] == response['journeys'][0]['sections'][0]['duration']
+        assert response['journeys'][0]['departure_date_time'] == response['journeys'][0]['sections'][0]['departure_date_time']
+        assert response['journeys'][0]['arrival_date_time'] == response['journeys'][0]['sections'][0]['arrival_date_time']
```

Comment:
could you add a test checking that the duration is 50% greater than with a speed factor=1 ?